assets/dev/js/editor/controls/popover-toggle.js

```diff
@@ -24,6 +24,12 @@ export default class ControlPopoverStarterView extends ControlChooseView {
 		if ( $popover.length ) {
 			$popover[ 0 ].dataset.popoverToggle = `elementor-control-default-${ this.model.cid }`;
 		}
+
+		$popover.on( 'hide', () => this.onPopoverHide() );
+	}
+
+	onPopoverHide() {
+		this.reRoute( false );
```

Comment:
What does the term `reRoute` means? and why is it needed?

Reply:
reRoute is a method for all controls to call the current route again but with the current control either active or inactive as passed in the argument. I originally had two methods `routeWithControlActive` and `routeWithControlInactive` but I combined them into one. Let me know if you have an idea for a better name maybe

Reply:
Talked about it with @Nevoss, waiting for resolve.